app/assets/stylesheets/print/print.scss

```diff
@@ -0,0 +1,5 @@
+// PRINT STYLESHEET COMPILER
+
+.print-wrapper {
```

Comment:
Can the compile live at `print.css` rather than `print/print.css`, I'd expect only the includes to be within the print directory.